Author: Andreas Beckmann <anbe@debian.org>
Description: set default device permissions to root:video 0660
 NVIDIA by default creates the device nodes /dev/nvidia* with owner
 root:root (0:0) and permissions rw-rw-rw- (0660).
 In Debian we have traditionally set them to root:video (0:44) and
 rw-rw---- (0660) via /etc/modprobe.d/nvidia-kernel-common.conf, but
 that no longer seems to work in all cases due to renaming the module
 for co-installability of current and legacy drivers.
 So let's change the built-in defaults instead, relaxing the permissions
 can be done via module options, e.g. in nvidia-kernel-common.conf.
Bug-Debian: https://bugs.debian.org/801598

--- a/nv-reg.h
+++ b/nv-reg.h
@@ -456,8 +456,8 @@ NV_DEFINE_REG_ENTRY(__NV_RESMAN_DEBUG_LE
 NV_DEFINE_REG_ENTRY(__NV_RM_LOGON_RC, 1);
 NV_DEFINE_REG_ENTRY(__NV_MODIFY_DEVICE_FILES, 1);
 NV_DEFINE_REG_ENTRY(__NV_DEVICE_FILE_UID, 0);
-NV_DEFINE_REG_ENTRY(__NV_DEVICE_FILE_GID, 0);
-NV_DEFINE_REG_ENTRY(__NV_DEVICE_FILE_MODE, 0666);
+NV_DEFINE_REG_ENTRY(__NV_DEVICE_FILE_GID, 44);
+NV_DEFINE_REG_ENTRY(__NV_DEVICE_FILE_MODE, 0660);
 NV_DEFINE_REG_ENTRY(__NV_REMAP_LIMIT, 0);
 NV_DEFINE_REG_ENTRY(__NV_UPDATE_MEMORY_TYPES, ~0);
 NV_DEFINE_REG_ENTRY(__NV_INITIALIZE_SYSTEM_MEMORY_ALLOCATIONS, 1);
